resources/google/slack-logger.ts

```diff
@@ -0,0 +1,120 @@
+import * as gcp from '@pulumi/gcp';
+import * as pulumi from '@pulumi/pulumi';
+import { apiServices } from './api-services';
+import { provider } from './provider';
+import { project } from './config';
+
+const config = new pulumi.Config('slack');
+const name = 'flexi-soft';
+const slackAgentTag = 'v3.1.0';
+const channel = 'C05CQ64DGBE'; // #flexi-soft-notifications
+
+const topic = new gcp.pubsub.Topic(name, {}, { provider });
+
+const serviceAccount = new gcp.serviceaccount.Account(
+  name,
+  {
+    accountId: name,
+  },
+  { provider },
+);
+
+const service = new gcp.cloudrunv2.Service(
+  name,
+  {
+    name: `slack-logger-${name}`,
+    location: 'europe-west1',
+    description: `Slack logger – ${name}`,
+    template: {
+      serviceAccount: serviceAccount.email,
+      containers: [
+        {
+          image: `docker.io/bjerkbot/google-cloud-logger-slack:${slackAgentTag}`,
+          envs: [
+            {
+              name: 'SLACK_TOKEN',
+              value: config.requireSecret('bot-oauth-token'),
+            },
+            {
+              name: 'DEFAULT_CHANNEL',
+              value: channel,
+            },
+          ],
+        },
+      ],
+    },
+  },
+  { provider },
+);
+
+new gcp.eventarc.Trigger(
+  name,
+  {
+    location: 'europe-west1',
+    transports: [
+      {
+        pubsubs: [
+          {
+            topic: topic.name,
+          },
+        ],
+      },
+    ],
+    matchingCriterias: [
+      {
+        attribute: 'type',
+        value: 'google.cloud.pubsub.topic.v1.messagePublished',
+      },
+    ],
+    serviceAccount: serviceAccount.email,
+    destination: {
+      cloudRunService: {
+        service: service.name,
+        region: 'europe-west1',
+      },
+    },
+  },
+  { provider, dependsOn: apiServices },
+);
+
+new gcp.projects.IAMMember(
+  name,
+  {
+    project: project,
```

Comment:
I was unsure about this variable. We are here inserting the `project` config value (which is the project name?) and I suspect this is expecting a project id?

Reply:
It should be the project id

Reply:
How can we retrieve that?

Reply:
You are already referring to that.